builder/ebssurrogate/step_snapshot_volumes.go

```diff
@@ -68,13 +68,15 @@ func (s *StepSnapshotVolumes) snapshotVolume(ctx context.Context, deviceName str
 			ResourceType: aws.String("snapshot"),
 			Tags:         snapshotTags,
 		}
-
 		tagSpecs = append(tagSpecs, snapTags)
 	}
-
+	description := s.SnapshotDescription
+	if description == "" {
+		description = fmt.Sprintf("Packer: %s", time.Now().String())
```

Comment:
I would suggest leaving the description empty be default, if only not to change the current behaviour by default, but also as the proposed description does not add information to the snapshot itself.

Reply:
Thanks for the comment.
The fix is pushed.